Why running lldb with HomeBrew Python 2 installed produces ImportError and NameErrors? <body> I had upgraded codevim/code with HomeBrew to version 7.4.1952 on OS X 10.11.6. It installed python 2 as a dependency package (Summarized parts of output is shown by code.../code):  precode$ brew upgrade vim ... ==&gt; Installing vim dependency: python    /usr/local/Cellar/python/2.7.11: 4,949 files, 66.6M ... /code/pre  Now when I try to run codelldb/code, it produces ImportError and NameErrors:  precode$ lldb myExecutableFile Traceback (most recent call last):   File "&lt;string&gt;", line 1, in &lt;module&gt;   File "/Applications/Xcode.app/Contents/SharedFrameworks/LLDB.framework/Resources/Python/lldb/__init__.py", line 98, in &lt;module&gt;     import six ImportError: No module named six (lldb) target create "myExecutableFile" Traceback (most recent call last):   File "&lt;string&gt;", line 1, in &lt;module&gt; NameError: name 'run_one_line' is not defined Traceback (most recent call last):   File "&lt;string&gt;", line 1, in &lt;module&gt; NameError: name 'run_one_line' is not defined ... Current executable set to 'myExecutableFile' (x86_64). (lldb) /code/pre  This is my code$PATH/code in zsh shell:  precode/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin /code/pre  How can I solve these errors?  <answer249988> I installed codesix/code module with codepip/code python package manager and it solves the problem:  precode$ pip install six Collecting six   Using cached six-1.10.0-py2.py3-none-any.whl Installing collected packages: six Successfully installed six-1.10.0  $ lldb myExecutableFile (lldb) target create "myExecutableFile" Current executable set to 'myExecutableFile' (x86_64). (lldb) /code/pre 